XLS-0033d-multi-purpose-tokens/README.md

```diff
@@ -127,16 +127,36 @@ A set of flags indicating properties or other options associated with this **`MP
 | `lsfMPTCanTrade`    | `0x0010`   | If set, indicates that _individual_ holders can trade their balances using the XRP Ledger DEX or AMM.                                                                                                                                     |
 | `lsfMPTCanTransfer` | ️`0x0020`  | If set, indicates that tokens held by non-issuers may be transferred to other accounts. If not set, indicates that tokens held by non-issuers may not be transferred except back to the issuer; this enables use-cases like store credit. |
 | `lsfMPTCanClawback` | ️`0x0040`  | If set, indicates that the issuer may use the `Clawback` transaction to clawback value from _individual_ holders.                                                                                                                         |
-
-Except for `lsfMPTLocked`, which can be mutated via the `**MPTokenIssuanceSet**` transactions, these flags are **immutable**: they can only be set during the **`MPTokenIssuanceCreate`** transaction and cannot be changed later.
+Except for `lsfMPTLocked`, which can be mutated via the **`MPTokenIssuanceSet`** transaction, these flags are
+**immutable** and can only be set once, at issuance creation, using the **`MPTokenIssuanceCreate`** transaction.
 
 ###### 2.1.1.2.3. `Issuer`
 
 The address of the account that controls both the issuance amounts and characteristics of a particular fungible token.
 
 ###### 2.1.1.2.4. `AssetScale`
 
-An asset scale is the difference, in orders of magnitude, between a standard unit and a corresponding fractional unit. More formally, the asset scale is a non-negative integer (0, 1, 2, …) such that one standard unit equals 10^(-scale) of a corresponding fractional unit. If the fractional unit equals the standard unit, then the asset scale is 0.
+Every MPT asset has an off-ledger standard unit of account. For example, the standard unit of account for a USD
+stablecoin is, in theory, one dollar. However, an MPT issuer might prefer to make the smallest unit of an MPT something
+smaller than the asset's standard unit, to enable on-ledger fractionalization while adhering to the limitation that
+MPTs only support whole numbers (MPT units must be integers). For example, an issuer might want each MPT unit to
+represent one cent instead of a whole dollar. Using `AssetScale`, MPT issuers can represent this difference as the
+number of orders of magnitude between a standard unit and an MPT unit.
+
+More formally, the asset scale is a non-negative integer (`0`, `1,` `2`, …) such that one standard unit
+equals $10^{-scale}$ of a corresponding MPT unit.
```

Comment:
```suggestion
More formally, the asset scale is a non-negative integer (`0`, `1,` `2`, …) such that one standard unit
equals $10^{scale}$ of a corresponding MPT unit.
```

Think of XRP as a standard unit, with an MPT representing drops. `scale = 6`. One standard unit (1 XRP) equals $10^{scale}$ MPT units (drops).